.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,20 @@
-# Description
-<!-- What does this pull request (PR) do? Why is it necessary? -->
-<!-- Tell us about your new content, improvement, or fix! -->
-<!-- If you can, please add an image, or an animation "An image is worth a thousand words!" -->
-<!-- You can use https://www.cockos.com/licecap/ or similar to create animations -->
-<!-- You can also see a preview of the documentation changes you are submitting by clicking on "Details" to the right of the "Check the rendered docs here!" check on your PR.-->
-
-## Type of change
-<!-- Please delete options that are not relevant. -->
-- [ ] Fixes or improves existing content
-- [ ] Adds new content page(s)
-- [ ] Fixes or improves workflow, documentation build or deployment
+# References and relevant issues
+<!-- What relevant resources were used in the creation of this PR?
+If this PR addresses an existing issue on the repo,
+please link to that issue here as "Closes #(issue-number)". -->
```

Comment:
Maybe worth adding that if it's docs for a napari PR, then using `depends on <napari PR link>` is helpful.

Reply:
Done, thanks!